SSM/GameManagers/GameManager.java

```diff
@@ -58,255 +64,27 @@ public class GameManager implements Listener, Runnable {
     private static short state = GameState.LOBBY_WAITING;
     private static long time_remaining_ms = 0;
     private static long last_time_update_ms = 0;
-    private static World lobby_world = Bukkit.getWorlds().get(0);
-    public static List<SmashGamemode> all_gamemodes = new ArrayList<SmashGamemode>();
     private static SmashGamemode selected_gamemode = null;
     private static MapFile selected_map = null;
 
     public GameManager() {
         all_gamemodes.add(new SoloGamemode());
         all_gamemodes.add(new TeamsGamemode());
         all_gamemodes.add(new TestingGamemode());
-        for(SmashGamemode gamemode : all_gamemodes) {
+        for (SmashGamemode gamemode : all_gamemodes) {
             gamemode.updateAllowedMaps();
             gamemode.updateAllowedKits();
             gamemode.createDataFiles();
         }
-        setGamemode(all_gamemodes.get(2));
+        setGamemode(all_gamemodes.get(0));
         Bukkit.getServer().getPluginManager().registerEvents(this, plugin);
         ourInstance = this;
         players.addAll(Bukkit.getOnlinePlayers());
-        for (Player player : players) {
-            player.teleport(lobby_world.getSpawnLocation());
-        }
+        SSM.teleportAllPlayers();
         Bukkit.getScheduler().scheduleSyncRepeatingTask(plugin, this, 0L, 0L);
         setState(GameState.LOBBY_WAITING);
     }
 
-    public void run() {
```

Comment:
Why did you move this all? Makes it impossible to see what you actually changed

Reply:
my intellij ducked up and it just messed up my game manager i just add the spawnFireworks at the end